tuna/miopen/db/session.py

```diff
@@ -38,6 +38,8 @@ class Session(BASE, SessionMixin):
   """Session table to keep track of tunning sesions"""
   #pylint: disable=attribute-defined-outside-init
   #pylint: disable=too-many-instance-attributes
+  #pylint: disable=no-name-in-module
+  #pylint: disable=no-value-for-parameter
```

Comment:
Do you why the linter is complaining about these?

Reply:
I am also curious what these 2 are about

Reply:
I think they were added to solve other errors. Now I see no dependency on these Pylint warning.